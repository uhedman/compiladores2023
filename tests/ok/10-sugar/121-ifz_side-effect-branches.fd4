let shouldPrintThen : Nat =
    1 + ifz 0 then print "Caso correcto" 0 else print "Caso incorrecto" 1

let shouldPrintElse : Nat =
    1 + ifz 1 then print "Caso incorrecto" 0 else print "Caso correcto" 1
