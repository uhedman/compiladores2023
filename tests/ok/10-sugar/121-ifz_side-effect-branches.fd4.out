Caso correcto0
Caso correcto1
